test/data/bad/feature_changed_semantics.result

```diff
@@ -0,0 +1 @@
+[{"message":"Feature object cannot contain features or coordinates","line":1}]
```

Comment:
How about "Feature object cannot contain a 'features' or 'coordinates' member"?

Once the draft is finished, we could add links for all of these.